Dmitriy_Vysotskyy2.0/WebDriverFactory.cs

```diff
@@ -0,0 +1,22 @@
+using System.Data;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+
+namespace Dmitriy_Vysotskyy2._0;
+
+public class WebDriverFactory
+{
+    public static IWebDriver GetWebDriver(BrowserType browserType)
```

Comment:
Let's put this and all related file to Driver folder
![image](https://github.com/dimavys/Dmitriy_Vysotskyy2.0/assets/47978115/4878c332-14f6-4e7b-a03d-ff3783c5f8a4)